WordPress/WordPress.xcodeproj/xcshareddata/xcschemes/WordPressIntents.xcscheme

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Folks, I'm not sure if this scheme should be shared or not.  IMHO there's no reason why schemes shouldn't be shared but if either of you knows better, can you let me know?

Reply:
I'm not sure either, although it appears we don't currently share any schemes (although it's also not in our gitignore)... @jkmassel any ideas?